lib/commands/download.js

```diff
@@ -15,10 +15,12 @@ module.exports = {
   description: 'Download i18n files from Crowdin',
 
   run: function(commandOptions) {
-    let config = loadConfig(this.project.root);
+    commandOptions = commandOptions || {};
+    const project = commandOptions.project || this.project;
```

Comment:
`this.project` is always the consuming application.  So we need to allow an addon to pass itself as the project instead, when calling this method